tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -274,4 +304,23 @@ static void setTrace(Trace trace) {
         MDC.put(Tracers.TRACE_ID_KEY, trace.getTraceId());
     }
 
+    protected static void checkForRemotingTracerCompatibility(Optional<OrderableSlsVersion> remotingTracerVer) {
```

Comment:
I think it's pretty important that this can be turned off if there's some exceptional reason why remoting3 tracing must still exist on the same classpath.